### ISSUE TITLE
Uncertainty evaluation

### ISSUE THREAD

**Issue by john-mai-2605** [Seondong/Customs-Fraud-Detection#4]
Make self-supervsied algorithm for uncertainty evaluation
Input: A tuple **data**, destructable into (train_loader, valid_loader, test_loader, leaf_num, importer_size, item_size, xgb_validy, xgb_testy, revenue_valid, revenue_test)

Output:   A list of score with the same length as test_loader.datasets.tensors[-1].shape[0] (test set size)
